### PR TITLE
Update wrapper with the latest client library changes

### PR DESCRIPTION
**Pull request by DmitryPR**
This adds support for Swift 4.2 and also some wrapper methods are updated to work with the client library

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Example
```
Comment (ostap0207) [inline .npmignore]: Don't we already have example somehwere?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (ostap0207) [inline package.json]: What does this commit name mean?

Comment (ostap0207) [reply package.json]: Please, make the commit name more clear.

**Comment by DmitryPR** [comment]
!merge

**Pull request merged**